Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py

```diff
@@ -92,12 +91,12 @@ def build_query_params(params):
     return '?' + query_params if query_params else ''
 
 
-def do_auth(server, crads, insecure):
+def do_auth(server, crads, insecure, domain):
     url = fix_url(str(server)) + 'sepm/api/v1/identity/authenticate'
     body = {
-        'username': crads.get('identifier') if crads.get('identifier') else '',
-        'password': crads.get('password') if crads.get('password') else '',
-        'domain': crads.get('domain') if crads.get('domain') else ''
+        'username': crads.get('identifier', ''),
+        'password': crads.get('password', ''),
```

Comment:
if `identifier` exists with a value `None` it will fail.
same for `password` and `domain`